app/main/views/marketplace.py

```diff
@@ -118,7 +116,11 @@ def list_opportunities(framework_framework):
 
     api_result = form.get_briefs()
 
-    briefs = api_result["briefs"]
+    briefs = [{
```

Comment:
I searched for 'lotName' in the frameworks repo but didn't find it included in any display manifests so not quite sure where to look. I may have to ask one of the catalogues team but it seems like a potential solution.

Based on our required fields for this view, using 'fake questions', I can see how we could drive it by a new manifest just for this page for
`id`, `title`, `summary`, `organisation`, `location`, `lotName`, `specialistRole` but I don't know how it would handle some of the logic on the page using the brief's status.

```
{% if brief.status == 'closed' %}
            <li class="search-result-metadata-item">
                Closed
            </li>
        {% else %}
            <li class="search-result-metadata-item">
                Published: {{ brief.publishedAt|dateformat }}
            </li>
            <li class="search-result-metadata-item">
                Closing: {{ brief.applicationsClosedAt|dateformat }}
            </li>
        {% endif %}
```
I don't think the content loader can do the logic around brief status at the moment, which means we are still passing in the brief and the content summary. The content loader also doesn't have a way of formatting these datetimes (we have only a `Date` question rather than a `DateTime` question). We could go to the effort of adding new question types for these fields which can do this logic and parse datetimes but I'm not sure if that feels like a good idea or not...